api/provider/aws/service_create.go

```diff
@@ -1,8 +1,137 @@
 package aws
 
-import "github.com/quintilesims/layer0/common/models"
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/quintilesims/layer0/common/models"
+)
 
 func (s *ServiceProvider) Create(req models.CreateServiceRequest) (*models.Service, error) {
+	var (
+		cluster          string
+		desiredCount     int
+		loadBalancer     *ecs.LoadBalancer
+		loadBalancerRole string
+		serviceName      string
+		taskDefinition   string
+	)
+
+	fqEnvironmentID := addLayer0Prefix(s.Config.Instance(), req.EnvironmentID)
+	cluster = fqEnvironmentID
+
+	desiredCount = 1
+
+	serviceID := generateEntityID(req.ServiceName)
+	fqServiceID := addLayer0Prefix(s.Config.Instance(), serviceID)
+	serviceName = fqServiceID
+
+	taskDefinitionARN, err := s.lookupTaskDefinitionARNFromDeployID(req.DeployID)
+	if err != nil {
+		return nil, err
+	}
+
+	taskDefinition = taskDefinitionARN
```

Comment:
`taskDefinitionARN` is more exact - why rename/replace here?

Reply:
Just an attempt to make variable names line up between the master function and the helper function. I'm happy to not rename. ✅